src/main/java/dev/th3shadowbroker/ouroboros/mines/listeners/BlockBreakListener.java

```diff
@@ -120,7 +120,15 @@ public void onBlockBreak(BlockBreakEvent event) {
     }
 
     private void breakBlock(BlockBreakEvent event, MineableMaterial mineableMaterial, ItemStack tool) {
-        if (!autoPickup) {
+        if(autoPickup || event.getPlayer().hasPermission("ouroboros.mines.autopickup")){
```

Comment:
Using `or` instead of an `and` might lead to unexpected behaviour. The user should have to explitly activate this feaure.
At the moment, the code as it is could cause false issue-reports.

```suggestion
        if(autoPickup && event.getPlayer().hasPermission("ouroboros.mines.autopickup")){
```

Reply:
Opted to go for the default permission for 'ouroboros.mines.autopickup = false' in plugin.yml - I think this adds more versatility, either enabled globally for all players or only enabled for those with the adequate permissions

Reply:
Alright! Thank you for your contribution 😄 
I'm currently doing a rework of the drop system.
In case there should be a merge-conflict after said rework, I'll resolve them, so your changes can be delivered with one of the next builds 🔧